src/fullcalendar/eventRender.js

```diff
@@ -29,6 +29,16 @@
  * @param {Node} el The HTML element
  */
 export default function({ event, el }) {
+	if (el.classList.contains('fc-event-nc-alarms')) {
+		const notificationIcon = document.createElement('span')
+		notificationIcon.classList.add('icon-event-reminder')
+		if (el.style.color === 'rgb(0, 0, 0)') {
```

Comment:
This will break, as soon we decide to change the [text colors](https://github.com/nextcloud/calendar/blob/master/src/utils/color.js#L47), which we probably will at some point since `#000000` is not the default text color in Nextcloud.

Furthermore I'm not sure that you can rely on the browser returning `'rgb(0, 0, 0)'` here. Others might return `'#000000'` or `'rgba(0, 0, 0, 0)'` or `'black'`.

Instead, you should add the information whether or not to use the dark icon in `event.extendedProps`

Reply:
Couldn't you use the svg icon api similar to https://github.com/nextcloud/tasks/blob/master/css/src/style.scss#L1174.

Reply:
I've modified the check to use a `darkText` property in `event.extendedProps`. I've also refactored the color helper functions a bit, since I needed a general function for detecting if a color was light or dark.